client/src/extension.ts

```diff
@@ -75,9 +75,7 @@ export function activate(context: ExtensionContext) {
 			};
 			let clientOptions: LanguageClientOptions = {
 				documentSelector:
-					fileSearchExtensions
-						.map(l => l.slice(1))
-						.concat(activeLanguages)
+					activeLanguages
```

Comment:
we also need to include the languages for files that we need to look through (i.e. style files like css, sass, less, etc.) so it can't just be `activeLanguages`. since we only support css, sass and less at the moment, perhaps we can just hard code it:

```
activeLanguages.concat(['css', 'sass', 'less']).map(...)
```

Reply:
Would it make sense if maybe we just included them in the activeLanguages setting default values list?
Then it's completely in the control of the user if they want those active

Reply:
Then we unnecessarily trigger peek lookups from within CSS files, when they should only happen in HTML, JSX, PHP, etc.

You could create a new configuration setting, but it kind of doesn't matter because even if someone add ".something", unless CSS_Peek knows how to parse ".something" style files, we can't do anything with it. 

I think hardcoding css, sass and less should be fine 😄

Reply:
Maybe I'm misunderstanding, if I am interpreting it right I have it set to only pull data from the include/exclude stuff and active languages controls which files has the peek stuff enabled

Reply:
Hey, not sure if you saw this ^ @pranaygp

Reply:
Yup, I think you got it right but I think the nomenclature is a bit confusing. Essentially, we need to separately handle:

1. files *from* which you can peek (html, php, ejs, etc.). This is configurable by the user
2. files *to* which you can peek (css, sass, less). This is *not* configurable by the user.

We didn't correctly handle it earlier because `searchFileExtensions` is actually redundant and shouldn't be able to be changed by the user I think. We can only support the VSCode "languages" for which we have parsers (i.e. css, sass and less)

What do you think?

Reply:
I realise now that this is actually quite different from your PR, but if you agree then it'd be a good opportunity now to remove that configuration, and just hard code the different services we provide when using globs (https://github.com/pranaygp/vscode-css-peek/pull/61/files/36f279e9a0d9c4bb798338ab3c20168039a4c0bd#diff-bc3c0f2fe3aca9ddbe96347ffb00641eR108)

Reply:
with your work we can still close #51 and #59. we'd still need to support the exclude option and have good defaults for it like `node_modules` :D

Reply:
I can give it a go

Reply:
@pranaygp check out the newest commit

Reply:
@pranaygp did you get a chance to check this out

Reply:
Thanks for the ping @arch-stack

Reply:
I've had a super busy couple weeks. I'll merge it in this weekend and test it all out before cutting a new version. Thanks a lot for the help @arch-stack!

If you'd like to help maintain `css-peek`, shoot me an email (pranay.gp@gmail.com)